src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -36,6 +36,7 @@ message NamespaceOptions {
     google.protobuf.BoolValue cacheBlocksOnRetrieve = 12;
     AggregationOptions aggregationOptions           = 13;
     google.protobuf.Any extendedOptions             = 14;
```

Comment:
nit: i think this is before extendedOptions is in use anywhere; can this be changed to like `99` or something? Just seems weird to have more regular options added after an extended opts

Reply:
That's an interesting idea. Never done that before. Are there any negative side effects to this (e.g. increased protobuf message size)?